src/ColorPickerComponent.js

```diff
@@ -1,46 +1,35 @@
-import React from 'react'
-import { SketchPicker } from 'react-color'
-import { rgb, hsl } from 'color-convert'
-import ColorGrid from './ColorGrid'
-
-class ColorPickerComponent extends React.Component {
-    state = {
-        displayColorPicker: false,
-        color: {
-            r: 241,
-            g: 112,
-            b: 19,
-        },
-    };
-
-    handleClick = () => {
-        this.setState( { displayColorPicker: !this.state.displayColorPicker } )
-    };
-
-    handleClose = () => {
-        this.setState( { displayColorPicker: false } )
-    };
-
-    handleChange = ( color ) => {
-        this.setState( { color: color.rgb } )
-    };
-    render() {
-        var colorAsHSL = rgb.hsl(this.state.color.r, this.state.color.g, this.state.color.b);
-        return (
-            <div className='main'>
-                <div >
-                    <ColorGrid
-                        h={colorAsHSL[0]}
-                        satCount="15"
-                        lumCount="15"
-                    />
-                </div>
-                <div className='colorpick'>
-                    <SketchPicker color={this.state.color} onChange={this.handleChange} />
-                </div>
-            </div>
-        )
-    }
-}
-
-export default ColorPickerComponent
\ No newline at end of file
+import React, { useState } from 'react';
+import { SketchPicker } from 'react-color';
+import { rgb } from 'color-convert';
+import ColorGrid from './ColorGrid';
+
+
+const ColorPickerComponent = () => {
+  const [displayColorPicker, setDisplayColorPicker] = useState(false);
+  const [color, setColor] = useState({ r: 241, g: 112, b: 19 });
+
+  const handleClick = () => setDisplayColorPicker((prev) => !prev);
+
+  const handleClose = () => setDisplayColorPicker(false);
+
+  const handleChange = (color) => setColor(color.rgb);
+
+  const colorAsHSL = rgb.hsl(color.r, color.g, color.b);
+
+  return (
+    <div className='main'>
+      <div>
+        <ColorGrid
+          hueCount={colorAsHSL[0]}
```

Comment:
its hueValue, not count.
satCount and lumCount were for the grid, and h was for the hue to work with, please update the name of hueCount to hueValue

Reply:
done